pkgs/development/python-modules/aiometer/default.nix

```
@@ -0,0 +1,38 @@
+{ lib
+, python3
+, fetchFromGitHub
+}:
+
+python3.pkgs.buildPythonPackage rec {
+  pname = "aiometer";
+  version = "0.4.0";
+  format = "pyproject";
+
+  src = fetchFromGitHub {
+    owner = "florimondmanca";
+    repo = "aiometer";
+    rev = version;
+    hash = "sha256-Db6AWNWZ5AhMFrbjBRVOFlDpzegsqNL3k5dmTHrjSi8=";
+  };
+
```

Something like this is likely needed:

```suggestion

  # disable coverage testing
  postPatch = ''
    substituteInPlace setup.cfg \
      --replace "--cov" "#--cov"
  '';

```